packages/manager/modules/pci/src/projects/project/storages/object-storage/users/add/user-add/user-add.controller.js

```diff
@@ -66,13 +66,28 @@ export default class PciUsersAddController {
         });
     }
     // action === 'createNewUser'
-    return this.createUser(user.description)
+    return this.createUser(this.newUserDescription)
       .then((data) => this.generateUserS3Credential(data))
       .finally(() => {
         this.isLoading = false;
       });
   }
 
+  isUserLinkedMode() {
+    return this.model === this.addExistingUser;
+  }
+
+  isUserCreationMode() {
```

Comment:
Imo this function name is lacking something, I don't understand did you mean 
```suggestion
  isUserCreationModeActive() {
```
?

Reply:
done